Formula/pkg-config-wrapper.rb

```diff
@@ -0,0 +1,19 @@
+class PkgConfigWrapper < Formula
+  desc "Easier way to include C code in your Go program"
+  homepage "https://github.com/influxdata/pkg-config"
+  url "https://github.com/influxdata/pkg-config/archive/v0.2.8.tar.gz"
+  sha256 "9d3f3bbcac7c787f6e8846e70172d06bd4d7394b4bcd0b8572fe2f1d03edc11b"
+  license "MIT"
+  head "https://github.com/influxdata/pkg-config.git", branch: "master"
+
+  depends_on "go" => :build
+  depends_on "pkg-config" => [:build, :test]
```

Comment:
Isn't it needed at runtime to actually use pkg-config though?

Reply:
it is a wrapper over pkg-config during the build time, that is why I did not specify it as runtime dependency. let me know what you think.

Reply:
But it uses pkg-config when you use it. So it's a runtime dependency.

Reply:
sounds good, let me move to runtime dependency then.